src/main/resources/db/migration/V3__tilkjent_ytelse.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE tilkjent_ytelse (
+    id                                  BIGSERIAL          primary key,
+    personIdent                         VARCHAR (20)       unique,
+    saksnummer                          VARCHAR(50)        unique,
```

Comment:
Vil personIdent og saksnummer være unike - hvis man har en verdi på `forrige_tilkjent_ytelse_id_fkey` ville jeg trodd at saksnummer var lik mellom disse to?

Reply:
Jo, det er riktig. Usikker på hvordan dette skal modelleres, og må nok sees opp mot hvilke andre konsepter som skal være på plass, type (Fag-)Sak, Behandling, Vedtak etc. 

Ville være på den sikre siden enn så lenge